mountpoint-s3-client/src/object_client.rs

```diff
@@ -632,6 +632,9 @@ impl UploadChecksum {
 /// Maintain a smaller window to limit the amount of data buffered in memory.
 pub trait ClientBackpressureHandle {
     /// Increment the flow-control read window, so that response data continues downloading.
+    ///
+    /// The client should read data that is included within the read window,
```

Comment:
is this comment meant to convey that client may return more data than expected since the read window is always aligned to the `read_part_size`?

Reply:
No, this comment was worded a bit badly. I just want to convey that if you were to send 1 byte here, it would trigger an 8MiB GetObject (or whatever the part size is). I'll reword!